container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -259,6 +259,10 @@ function generate_test_results {
 
 export OS_CLOUD=default
 
+if [ ! -z ${TEMPEST_OCP_CLIENT_TGZ} ]; then
```

Comment:
I'm wondering whether we want to install the oc cli by default in all tempest images or only in the tempest-all images. The installation is quite quick.